### DISCUSSION TITLE
Synchronous communication channel

### DISCUSSION THREAD

**Discussion by keunes** [OpenPodcastAPI/api-specs#1] [General / Meta]
During the meeting it was suggested that a synchronous text channel would be helpful to support asynchronous work and live discussions. Some of the suggestions were Slack, Matrix (Element), Mattermost. So, what should we go for?

**Comment by georgkrause** [comment]
At Funkwhale we are pretty happy using Mattermost, but this is mostly because it integrates pretty well with our services and brings a lot of project management features. I am not sure if this applies here, since we are not having resources for self hosting from the beginning and we seem to have Github for the project management anyways. Slack is a no-go from my perspective for any serious opensource project, so I'd like to use Matrix: Its open and easy and featurefull.

**Comment by genebean** [comment]
I wasn’t involved, but this seems like a great use case for Matrix to me.

**Comment by georgkrause** [comment]
@keunes How do we proceed here? I think we should just start instead of waiting any longer for more hands :)

**Reply by keunes** [reply]
Hi @georgkrause,
You're absolutely right. I was focussing on other things for a while and let this slide s bit. Thanks for pinging :-)

I just created a room: https://matrix.to/#/!ZHdcrdWSgxXRREuJdU:matrix.org?via=matrix.org

Can folks join with the current privacy settings?

**Reply by georgkrause** [reply]
I get "You are not invited"

**Reply by keunes** [reply]
Ok, changed the setting from Private to Anyone can find and join the room. Hoping it works now, while not being findable in the matrix.org repository.

**Reply by genebean** [reply]
That works for me

**Reply by keunes** [reply]
@bohwaz I saw you're making a minimalist gpodder server, and figured you might be interested in the developments here. The draft for Subscriptions sync was just released. You're more than welcome also to join the Matrix channel!

**Reply by bohwaz** [reply]
> @bohwaz I saw you're making a minimalist gpodder server, and figured you might be interested in the developments here. The draft for Subscriptions sync was just released. You're more than welcome also to join the Matrix channel!

Thank you! I don't have much time to follow the discussions sorry, the server is here: https://github.com/bohwaz/micro-gpodder-server

It's using a small part GPodder API, enough to sync episodes and subscriptions that's all.

Sadly I don't remember much about that, but I had some issues with the API when I implemented it, if I remember them I'll report them.

Anyway, thanks for mentioning me, and thanks to all of you for the time and energy spent on that, it's great!
